jenkins/release-workflows/release-notes-generate.jenkinsfile

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+lib = library(identifier: 'jenkins@10.2.2', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+pipeline {
+    options {
+        timeout(time: 2, unit: 'HOURS')
+    }
+
+    agent none
+
+    parameters {
+        string(
+            name: 'INPUT_MANIFEST',
+            description: 'Release version number to fetch input manifest path',
```

Comment:
Description and parameters seems to be mismatched. Let's just take version number from the user.

Reply:
Would keep it as manifest and fix description, as taking in version will mean I need to process OS and OSD in the same run, spawning ~50 agents in a single run. I want to be flexible to be able to process one product at a time.